src/main/java/seedu/application/logic/parser/ParserUtil.java

```diff
@@ -189,34 +198,4 @@ public static InterviewAddress parseInterviewAddress(String interviewAddress) th
         }
         return new InterviewAddress(trimmedInterviewAddress);
     }
-
-    /**
-     * Parses a {@code String interviewPreamble} into an array of strings.
-     * Leading and trailing whitespaces will be trimmed, and the string will be split by spaces.
-     *
-     * @throws ParseException if the given {@code interviewPreamble} is invalid or cannot be parsed.
-     */
-    public static String[] parseInterviewPreamble(String interviewPreamble) throws ParseException {
-        requireNonNull(interviewPreamble);
-        String[] trimmedInterviewPreamble = interviewPreamble.trim().split(" ");
-        if (trimmedInterviewPreamble.length != 2) {
-            throw new ParseException(InterviewCommand.MESSAGE_CONSTRAINTS);
-        }
-        return trimmedInterviewPreamble;
-    }
-
-    /**
-     * Parses a {@code String subCommandWord} into a String.
-     * Leading and trailing whitespaces will be trimmed.
-     *
-     * @throws ParseException if the given {@code subCommandWord} is not a valid sub-command.
-     */
-    public static String parseSubCommandWord(String subCommandWord) throws ParseException {
-        requireNonNull(subCommandWord);
-        String trimmedSubCommandWord = subCommandWord.trim();
-        if (!InterviewCommand.isValidSubCommandWord(trimmedSubCommandWord)) {
-            throw new ParseException(InterviewCommand.MESSAGE_CONSTRAINTS);
-        }
-        return trimmedSubCommandWord;
-    }
```

Comment:
What were these methods used for previously? Can't really tell why they were deleted here. Were they not used?